src/Shared/CommunicationsUtilities.cs

```diff
@@ -925,16 +924,24 @@ internal static HandshakeOptions GetHandshakeOptions(
             {
                 context |= HandshakeOptions.NodeReuse;
             }
+
             if (lowPriority)
             {
                 context |= HandshakeOptions.LowPriority;
             }
+
 #if FEATURE_SECURITY_PRINCIPAL_WINDOWS
```

Comment:
#if FEATURE_SECURITY_PRINCIPAL_WINDOWS is set only for
<img width="1330" height="49" alt="image" src="https://github.com/user-attachments/assets/71f79221-2032-4d94-88e2-89709b5d47a5" />